### ISSUE TITLE
Cannot leave network settings panel

### ISSUE THREAD

**Issue by GoogleCodeExporter** [maximwhere/telephone#273]
```
What steps will reproduce the problem?
1. Open settings window
2. Switch to network settings tab
3. Try to close settings window or try to select other tab

What is the expected output? What do you see instead?

It pops up a sheet asking if I want to save the changes to the networking 
settings, even if I didn't 
change any settings. If I click don't save or save the sheet just reappears 
blocking me from 
closing the settings window or changing the active tab. If I click save the 
status in the main 
window changes to offline and back to available.

The save networking settings sheet should only pop up if I actually did change 
something and it 
should appear only once allowing me to close the window or change tabs after I 
chose to save or 
not to save those settings.

What version of the product are you using? On what operating system?

Telephone 0.14.3 (23) on Snow Leopard 10.6.2

Please provide any additional information below.

I am using the german translation of Telephone.

```

Original issue reported on code.google.com by `sven.weidauer@gmail.com` on 14 Jan 2010 at 2:37
- Merged into: #232
